### DISCUSSION TITLE
disable with_drag_and_drop in glium to prevent crashing in certain apps

### DISCUSSION THREAD

**Discussion by AyashiNoCeres** [emilk/egui#291] [Q&A]
I have a requirement that I need to set window_builder with with the following setting while using eframe.

```
window_builder = glium::glutin::window::WindowBuilder::new()
            .with_drag_and_drop(false)
```

Basically to solve an issue like this where my program crashes if drag_and_drop is true

https://github.com/mvdnes/rboy/pull/10/commits/ff7c3c27d11cbf53dfd6dfccdd738062020537bf

How do I do that in egui?

**Comment by emilk** [comment]
I don't see `with_drag_and_drop` as an option at https://docs.rs/glutin/0.26.0/glutin/?search=with_drag_and_drop
